components/metadata-service/internal/metadata/remoteenv/util.go

```diff
@@ -103,7 +103,7 @@ func replaceService(id string, re *v1alpha1.RemoteEnvironment, service v1alpha1.
 
 func ensureServiceExists(id string, re *v1alpha1.RemoteEnvironment) apperrors.AppError {
 	if !serviceExists(id, re) {
-		message := fmt.Sprintf("Service with ID %s doesn't exist.", id)
+		message := fmt.Sprintf("Remote Environment: exitting of service with ID %s failed", id)
```

Comment:
The meaning of this error message is unclear.

Reply:
Ok, I'll take care of it.